os/fs/smartfs/smartfs_smart.c

```diff
@@ -1663,7 +1663,6 @@ static int smartfs_statfs(struct inode *mountpt, struct statfs *buf)
 	smartfs_semtake(fs);
 
 	/* Implement the logic!! */
-
```

Comment:
This is an unnecessary change. Let's revert.

Reply:
Done.